SU-SWS/sulgryphon-nextjs
# Sul23 434

# READY FOR REVIEW

# Summary
- The `stanford-lists` component needed to be made aware of the shared tags event view we set up here: https://github.com/SU-SWS/ace-stanfordlagunita/pull/98

# Review By (Date)
- 6/5/2024, prior to release

# Urgency
- the new view won't work without it.

# Steps to Test

1.  I tested using the branner-earth-sciences-library-map-collections location, since they have a card set up for events using the old shared-tags view.  
2. Create a new event if necessary, scheduled at some point in the future.  
3. When you set up the event, added the Branner Library location in the taxonomy fields.
4. in the Branner LIbrary location page, find the Events list card.  In that card, change the view from `"Shared Tags" Content` to `Shared Tag Events`.  No other change should be necessary.
5. On the front end, make sure that only events which are scheduled today or later are pulled in to the list card, and make sure it still looks correct. 

# Affected Projects or Products
- no

# Associated Issues and/or People
- https://github.com/SU-SWS/ace-stanfordlagunita/pull/98

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
